Resources/public/Admin.js

```diff
@@ -103,6 +103,31 @@ var Admin = {
                 checkboxClass: 'icheckbox_square-blue',
                 radioClass: 'iradio_square-blue'
             });
+
+            var previousIndex;
+
+            jQuery('tbody .iCheck-helper', subject).click(function (evt) {
```

Comment:
It's bound to the iCheck "decorator" (the visible checkbox), not the input itself as it's not directly clicked.